tools/run_tests/performance/README.md

```diff
@@ -340,20 +348,38 @@ was generated from the example configurations in
 
 ```
 $ ./tools/run_tests/performance/loadtest_template.py \
-    -i ../test-infra/config/samples/*.yaml \
-    --inject_client_pool --inject_server_pool --inject_big_query_table \
-    --inject_timeout_seconds \
+    -i ../test-infra/config/samples/*_example_loadtest.yaml \
+    --inject_client_pool --inject_driver_pool --inject_server_pool \
+    --inject_big_query_table --inject_timeout_seconds \
     -o ./tools/run_tests/performance/templates/loadtest_template_basic_all_languages.yaml \
     --name basic_all_languages
 ```
 
+The example template with prebuilt images in
+[loadtest_template_prebuilt_all_languages.yaml](./templates/loadtest_template_prebuilt_all_languages.yaml)
+was generated by the following command:
+
+```
+$ ./tools/run_tests/performance/loadtest_template.py \
+    -i ../test-infra/config/samples/*_example_loadtest_with_pre_built_workers.yaml \
```

Comment:
nit: suggestion for consistency `-i ./config/samples/*_example_loadtest_with_pre_built_workers.yaml \`

Reply:
The input for the templates is coming from the test-infra repository. Later on we use the template (now in the grpc repo) to generate configs. The  paths in this example and in the loadtst_config.py example are different for this reason.

Reply:
I see, thanks for explaining.